### PR TITLE
Seeing all spaces

### PR DESCRIPTION
**Pull request by breadmaking**
Self.all method has been created.
Initialize method created.
Refactoring done.
Database setup done.
Tests done.
Integrated Bootstrap CSS.

### CODE REVIEW

File: lib/space.rb [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+      Space.new(title: space['title'])
+    end
+  end
+
+  private 
+
+  def self.connection
+    if ENV['ENVIRONMENT'] == 'test'
+      conn = PG.connect(dbname: 'makersbnb_test')
+    elsif conn = PG.connect(dbname: 'makersbnb')
```
Comment (harmlessgoose) [inline lib/space.rb:37]: Maybe this could be an else statement?

File: spec/spec_helper.rb [hunk 2]
```diff
@@ -1,21 +1,6 @@
...
-# will add to the boot time of your test suite on EVERY test run, even for an
-# individual file that may not need all of that loaded. Instead, consider making
-# a separate helper file that requires the additional dependencies and performs
-# the additional setup, and require it from the spec files that actually need
-# it.
-#
-# See https://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
-
 # Set the environment to "test"
 ENV['RACK_ENV'] = 'test'
```
Comment (now-what) [inline spec/spec_helper.rb:2]: this one can go then?

File: spec/spec_helper.rb [hunk 3]
```diff
@@ -1,21 +1,6 @@
...
+ENV['ENVIRONMENT'] = 'test'
```
Comment (harmlessgoose) [inline spec/spec_helper.rb:3]: Both these lines are doing the same thing, the second one could be removed

**Review by harmlessgoose — CHANGES_REQUESTED** [review]
Couple of lines to amend, but looks great!

File: spec/unit/space_spec.rb [hunk 4]
```diff
@@ -0,0 +1,22 @@
+require 'space'
+require 'pg'
```
Comment (now-what) [inline spec/unit/space_spec.rb:2]: I think because you're already requiring `pg` in `space.rb`, then there's no need to require it here.

**Review by now-what — CHANGES_REQUESTED** [review]

**Pull request merged**